modules/accredited_representative_portal/app/controllers/accredited_representative_portal/v0/representative_form_upload_controller.rb

```diff
@@ -19,19 +19,59 @@ def submit
 
       def upload_scanned_form
         authorize(nil, policy_class: RepresentativeFormUploadPolicy)
-        attachment = PersistentAttachments::VAForm.new
-        attachment.form_id = params['form_id']
+        attachment = create_form(PersistentAttachments::VAForm)
+        validate(attachment)
+      end
+
+      def upload_supporting_documents
+        authorize(nil, policy_class: RepresentativeFormUploadPolicy)
+        attachment = create_form(PersistentAttachments::VAFormDocumentation)
+        validate(attachment)
+      end
+
+      private
+
+      def create_form(form_type)
+        attachment = form_type.new(form_id: params[:form_id])
         attachment.file = params['file']
-        raise Common::Exceptions::ValidationErrors, attachment unless attachment.valid?
+        attachment
+      end
 
-        attachment.save
-        serialized = PersistentAttachmentVAFormSerializer.new(attachment).as_json.deep_transform_keys do |key|
+      def validate(attachment = nil)
```

Comment:
Rename to `validate_attachment!` to indicate that this method relates to the attachment functionality in this file, rather than submission functionality.